code/game/machinery/bots/bots.dm

```diff
@@ -55,9 +55,10 @@
 	if(locked)
 		locked = 0
 		emagged = 1
-		to_chat(user, "<span class='warning'>You bypass [src]'s controls.</span>")
+		to_chat(user, "<span class='warning'>You remove [src]'s control restrictions. Swiping again will cause [src] to malfunction.</span>")
```

Comment:
Though I don't like anything too obviously immersion breaking I really can't think of an ingame description that would tell you to do this that idiots couldn't misinterpret.